### ISSUE TITLE
How to modified page title in <head>

### ISSUE THREAD

**Issue by jhyejun** [sergiokopplin/indigo#296]
i want to this

before
```
<head>
    <title>About | John Doe</title>
</head>
```

after
```
<head>
    <title>John Doe</title>
</head>
```

Maybe indigo using jekyll-seo-tag, how can fix it?

**Comment by johnwmillr** [comment]
This is not an ideal fix, but I was able to change the tab title by adding
```html
<title> NEW TAB NAME </title>
```
above the `{% seo %}` line in the `_layouts/default.html` file.

See [this commit](https://github.com/johnwmillr/johnwmillr.github.io/commit/50d425e9eb7dce99287611408963a7f4c0a4166d) for the code and [my website](http://www.johnwmillr.com/) for an example.

This fix is relevant to Issues #140 and #261 as well.

**Comment by jhyejun** [comment]
Thank you for help
